### DISCUSSION TITLE
Commit author name

### DISCUSSION THREAD

**Discussion by hellopavelzinoviev** [Vinzent03/obsidian-git#532] [Problems]
Hey! 

Can anyone help me find where I can set up the commit author name? The problem is that my name is misspelled in the commits in the remote repository, and I can't find where to correct it. I've already checked plugin's setting, global and local .gitconfig. Where else should I look?

**Comment by Vinzent03** [comment]
It's in the plugin settings under "Commit Author".

**Reply by hellopavelzinoviev** [reply]
Yeah, I've seen this field in the settings. And it filled in correctly (as I want my name to be displayed), but I still see different name in the github repo.

**Reply by Vinzent03** [reply]
Works fine for me in the local log output. What is the author when running `git log` in your repository? If it shows the correct author, GitHub may using a name from your account and not the author in your git config.

**Reply by hellopavelzinoviev** [reply]
`git log` command shows the correct name, as it written in the global and the local config file. And the name of my GitHub account also differs from the name of commits author name. In another words my name is the same in the all of the configs and settings, but it is different in the remote repo commits...

**Reply by Vinzent03** [reply]
For me it's just showing my username, but if the log output is correct then the issue is on GitHub and thus no issue with this plugin. You can verify this by simply manually commiting.
